docker/Dockerfile

```diff
@@ -1,24 +1,34 @@
-FROM centos:7 as stage1
+FROM ubuntu:22.04 as stage1
 
 COPY setup-hw /usr/local/bin/setup-hw
 COPY monitor-hw /usr/local/sbin/monitor-hw
 COPY collector /usr/local/sbin/collector
 COPY setup-apply-firmware /usr/local/sbin/setup-apply-firmware
 COPY setup-isoreboot /usr/local/sbin/setup-isoreboot
-
+COPY systemctl-dummy /usr/bin/systemctl
```

Comment:
I'm not sure whether this dummy is necessary in `stage1`.
If it is used only in the final stage, please move this COPY to that stage.

Note that `stage1` is not just an intermediate stage.
`stage1` is published as `quay.io/cybozu/setup-hw` while the final stage is `quay.io/cybozu/setup-hw-secret`.

Reply:
I move it in stage1. Thanks for your sugesstion.